reference/api/keys.md

```diff
@@ -68,22 +75,59 @@ List all existing API keys. **Expired keys are included in the response**, but d
       "createdAt": "2021-08-11T10:00:00Z",
       "updatedAt": "2021-08-11T10:00:00Z"
     }
-  ]
+  ],
+  "offset":0,
+  "limit":3,
+  "total":7
 }
 ```
 
+::: note
+API keys are displayed in descending order based on their `createdAt` date. This means that the most recently created keys appear first.
+:::
+
 ### Returned fields
 
+Returns API keys in an array called `results`, along with the following fields:
+
+#### `offset`
+
+The number of keys skipped over.
+
+#### `limit`
+
+The maximum number of keys to be returned by the request.
+
+#### `total`
+
+The total number of API keys that can be browsed.
+
+### The `results` array
+
+For each key, it returns:
+
+#### `name`
+
+A human-readable name for the key. Default value is `null`.
+
 #### `description`
 
 A description for the key. Default value is `null`.
 
+#### `uid`
+
+A [uuid v4](https://www.sohamkamani.com/uuid-versions-explained) to identify the API key. If not specified, it is automatically generated by Meilisearch.
+
 #### `key`
 
-An alphanumeric key value generated by Meilisearch on API key creation. Used for authorization when [making calls to a protected Meilisearch instance](/learn/security/master_api_keys.md#communicating-with-a-protected-instance).
+An alphanumeric key value generated by Meilisearch by hasing the `uid` and the master key on API key creation. Used for authorization when [making calls to a protected Meilisearch instance](/learn/security/master_api_keys.md#communicating-with-a-protected-instance).
 
 This value is also used as the `{key}` path variable to [update](#update-a-key), [delete](#delete-a-key), or [get](#get-one-key) a specific key.
 
+::: note
+Since `key` is a hash of the `uid` and master key, `key` values are deterministic between instances sharing the same configuration. This means if the master key changes, all key values are automatically changed and that API key resources are propagated into dumps and snapshots without displaying the `key` field in clear.
+:::
```

Comment:
Nice explanation here ✨ 
If a snapshot or a dump leaks into nature, nobody would be able to know the key value without knowing the master key. 👍

Reply:
Regarding this part @meilisearch/docs-team, it could be super cool to give users a bash command to know the `key` field regarding the master-key value and the uid. 

It could be useful for continuous deployment. See https://github.com/meilisearch/product/discussions/421#discussioncomment-3079410

The core team is looking at this to see what can be proposed!

Reply:
@gmourier, I added the bash command [here](https://github.com/meilisearch/documentation/pull/1738/files#diff-d40e36ee90d9727237e72205b40af7ea904d5d0ce0b277774e7c5ebe8501bdf3). Could you please review this part? 🙏